a1_intelligence-management-platform.html

```diff
@@ -1,285 +1,291 @@
+
+                <h2 class="bold-heading">Characteristics</h2>
+
+                <dl>
+                    <dt>Insightfulness</dt>
+                    <dd>Interrogate and understand your data, know what algorithms you have and make intelligent decisions</dd>
 
-<section class="default-bg-section">
-	<div class="edged-section u-text-center">
+                    <dt>Maximise</dt>
+                    <dd>Your ROI for all Artificial Intelligence initiatives</dd>
 
-        <h2 class="bold-heading">How it works</h2>
+                    <dt>Flexibility</dt>
+                    <dd>Designed for both Quantum and Classical computing algorithms</dd>
 
-        <img src="img/How-A1-Works.png" alt="The A1 Platform architecture diagram" class="u-centered-image">
-	</div>
-</section>
+                    <dt>Traceability</dt>
+                    <dd>Track & audit ML experiments, insights and intelligent data</dd>
 
-<section class="alternate-bg-section u-padding-top-medium u-padding-bottom-medium">
-	<div class="u-text-center">
+                    <dt>Trainability</dt>
+                    <dd>Our algorithm can learn the patterns in your data and apply insights through machine learning</dd>
 
-        <h2 class="bold-heading">User Experience</h2>
+                    <dt>Consistency</dt>
+                    <dd>Produce repeatable insights and experiments facilitating evolution and maximising accuracy</dd>
 
-        <div class="u-row -three-items-stackable">
-            <div>
-                <h3>C-level executive</h3>
-                <iframe width="96%" height="315" src="https://www.youtube.com/embed/48qkp3cpmb8" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+                    <dt>Utility</dt>
+                    <dd>Works with any machine learning library, language and code</dd>
+
+                    <dt>Reliability</dt>
+                    <dd>Runs the same way in any cloud, on-premise and bare metal deployment</dd>
+
+                    <dt>Scalability</dt>
+                    <dd>Built to scale and meet the demands of large geographically dispersed organisations, as they grow</dd>
+                </dl>
             </div>
+        </section>
+
+        <section class="emphasised-bg-section">
+            <div class="edged-section u-text-center">
+                <h2 class="default-heading">Why</h2>
+                <p>A1 Intelligence Management Platform enables you to quickly and effectively answer key questions, such as:</p>
+
+                <div class="u-row -four-items-stackable-in-two">
+                    <div>
+                        <h3>ROI</h3>
+                        <p>Do you know the business value generated from your AI/ML initiatives</p>
+                    </div>
+
+                    <div>
+                        <h3>Cost</h3 <p>How much are you spending on AI/ML in which functional areas</p>
```

Comment:
As you touched the line (whitespace) it drew my attention to the malformed closing of the h3 tag... Would you fix it please?

Reply:
I have updated this and also did change for Eric in this branch.